webhooks-extension/config/triggers_prototype/github/webhook.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: webhook-task
+  namespace: tekton-pipelines
+spec:
+  inputs:
+    params:
+    - name: Mode
+      description: Whether or not to create or delete the webhook
+      default: "create"
+    - name: ExternalUrl
+      description: The external access URL (used for creating the webhook)
+      default: "replacemeip.nip.io"
+    - name: GithubOwner 
```

Comment:
I wonder if this is GitHubOwnerOrOrgName rather than the Owner - could just change the description though??  Note - has this been tested against an org?

Also weird use of camel casing GitHub in descriptions but not on parameter names

Reply:
Replaced param names, it's owner yeah